packages/twenty-server/src/engine/api/graphql/workspace-query-runner/workspace-query-runner.service.ts

```diff
@@ -545,19 +555,25 @@ export class WorkspaceQueryRunnerService {
       options,
     );
 
-    // TODO START: remove this awful patch and use our upcoming custom ORM is developed
-    const deletedWorkspaceMember = await this.handleDeleteWorkspaceMember(
-      args.id,
-      workspaceId,
-      objectMetadataItem,
-    );
+    if (objectMetadataItem.standardId === STANDARD_OBJECT_IDS.workspaceMember) {
+      deletedWorkspaceMember = await this.workspaceMemberRepository.findOne({
+        where: { id: args.id },
+        select: { userId: true },
+      });
+    }
 
-    const deletedBlocklistItem = await this.handleDeleteBlocklistItem(
-      args.id,
-      workspaceId,
-      objectMetadataItem,
-    );
-    // TODO END
+    if (objectMetadataItem.standardId === STANDARD_OBJECT_IDS.blocklist) {
+      deletedBlocklistItem = await this.blocklistWorkspaceRepository.findOne({
```

Comment:
@magrinj I think we want to have any object added to the event, here we have hardcoded blocklist and workspaceMember as we had no other solution but we want this to be generic (this would require the orm to be based on metadata)